config_sample.yaml

```diff
@@ -20,6 +20,15 @@ redmine_url: <redmine_url>
 redmine_task_prefix: <redmine task prefix>
 redmine_username: <redmine api username/email>
 redmine_password: <redmine password>
+service:
+  sleep_time: 30
```

Comment:
(env) [root@okhatavk jirasync]# grep -ir 'Next run in' /var/log/messages
Aug  6 12:30:19 okhatavk jirasync: Next run in 30 seconds...

For SAT QE to complete the whole cycle takes almost 1-2 minute, 30 secs are very less. We should increase at least 180-300 secs. We have a team almost of 18-20 peoples.

Reply:
ok, I'll increase to `300` although this does not matter too much, the execution is blocking so the `sleep_time` is just an interval between runs, for example: It will run and take 1-2minute to complete, then it will sleep for 300 seconds, then it will run again and take 1-2 minute, then sleep again 300 seconds....